packages/zoe/src/internal-types.js

```diff
@@ -148,7 +151,7 @@
  * @param {ProposalRecord} proposal
  * @param {ExitObj} exitObj
  * @param {SeatHandle} seatHandle
- * @returns {ZoeSeatAdminKit}
+ * @returns {{userSeat: UserSeat, notifier: Notifier<Allocation>}}
```

Comment:
With this return type, we need to change the name from `*EscrowSeat` to `*EscrowSeatKit`

Reply:
done